packages/integrations/mdx/src/plugins.ts

```diff
@@ -112,6 +114,30 @@ export function rehypeApplyFrontmatterExport(pageFrontmatter: Record<string, any
 	};
 }
 
+/**
+ * `src/content/` does not support relative image paths.
+ * This plugin throws an error if any are found
+ */
+function remarkContentRelImageError() {
+	return (tree: any, vfile: VFile) => {
+		if (!vfile.path.includes('content/')) return;
+
+		const relImagePaths = new Set<string>();
+		visit(tree, 'image', function raiseError(node: Image) {
+			if (isRelativePath(node.url)) {
+				relImagePaths.add(node.url);
+			}
+		});
+		if (relImagePaths.size === 0) return;
+
+		const errorMessage =
+			`Relative image paths are not support in the content/ directory. Please update to absolute paths:\n` +
```

Comment:
It's not super clear what "absolute paths" means here... can we be more helpful?

Reply:
Ah, tough to be concise yet useful when we don't have the luxury of `AstroError`. I'll redraft

Reply:
I guess I'd just be a bit confused. Do we mean to put images in `public/` and use a base-relative absolute URL?

Reply:
Yep, `public` should be the recommendation here 👍